src/reducerFuncs/categoryreducer.jsx

```diff
@@ -0,0 +1,40 @@
+export const categories = [
+    {
+      _id: 1,
+      categoryName: "men",
+      description:
+        "Men's clothing",
+    },
+    {
+      _id: 2,
+      categoryName: "women",
+      description:
+        "Women's clothing",
+    },
+    {
+      _id: 3,
+      categoryName: "kids",
+      description:
+        "Kid's clothing",
+    },
+    {
+      _id: 4,
+      categoryName: "miscellaneous",
+      description:
+        "Micellaneous clothing",
+    },
+  ];
+
+  let catList = [];
+  categories.filter((item) => {
+      catList.push(item.categoryName);
+  })
```

Comment:
you don't need to make a new array catList you can directly use categories array

Reply:
since the categories array has a list of objects with category details, that's why i wanted to creat a separate array which i could use in other pages to iterate over jyst the category name. because i might still have use for other values from categories itself.